frontend/src/app/sessions/license-indicator/license-indicator.component.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { AsyncPipe, NgClass } from '@angular/common';
+import { Component } from '@angular/core';
+import { MatIcon } from '@angular/material/icon';
+import { T4CLicenseServerUsage } from '../../openapi';
+import { LicenseUsageWrapperService } from './license-usage.service';
+
+@Component({
+  selector: 'app-license-indicator',
+  standalone: true,
+  imports: [MatIcon, AsyncPipe, NgClass],
+  templateUrl: './license-indicator.component.html',
+})
+export class LicenseIndicatorComponent {
+  constructor(public licenseUsageWrapperService: LicenseUsageWrapperService) {}
+
+  getLevel(usage: T4CLicenseServerUsage) {
+    const usagePercentage = (usage.total - usage.free) / usage.total;
+    return this.levels.find(
+      (level) => usagePercentage * 100 >= level.percentage,
+    );
+  }
+
+  levels = [
+    {
+      percentage: 100,
+      text: 'All TeamForCapella licenses are currently in use. You can start new sessions, but may encounter the error "Invalid license" when trying to use TeamForCapella. Please make sure to terminate your sessions after use.',
+      icon: 'error',
+      classes: 'bg-red-500 text-white',
+    },
+    {
+      percentage: 75,
+      text: 'Most TeamForCapella licenses are currently in use. Please make sure to terminate your sessions after use.',
+      icon: 'warning',
+      classes: 'bg-yellow-300',
+    },
+    {
+      percentage: 0,
+      text: 'TeamForCapella licenses are available. You can start new sessions without any issues.',
```

Comment:
```suggestion
      text: 'TeamForCapella licenses are available. We do not expect any license issues in the session.',
```

"You can start new sessions without any issues." is a rather unspecific statement and doesn't necessarily limit the scope of the statement to licenses. I'd clarify it and be more specific.